### ISSUE TITLE
Flag `--check-defined` seems broken

### ISSUE THREAD

**Issue by wytseoortwijn** [utwente-fmt/vercors#105]
VerCors gives a `java.lang.NullPointerException` when using the `--check-defined` option. This already happens in the following program:
```java
class Future {
	process a(); 
}
```

**Comment by wytseoortwijn** [comment]
The `NullPointerException` seems due to a missing modifies clause. If the program is changed as follows, the exception no longer occurs
```java
class Future {
	int x;
	
	modifies x;
	process incr(); 
}
```

However, VerCors then gives another exception saying `java.lang.Error: condition class vct.col.ast.OperatorExpression without origin` on this example. This is with the command options `vct --silicon --check-defined`.

**Comment by wytseoortwijn** [comment]
Moreover, VerCors seems to have an `--infer-modifies` option, although it does not seem to do anything in this case. Perhaps we should look at this later...

**Comment by wytseoortwijn** [comment]
Note, the `./examples/histories/History.pvl` example file states that _"Checking the correctness of the defined processes and/or axioms is disabled due refactoring"_. This statement applies on the `--check-defined` and `--check-axioms` flags.

**Comment by FauziaE** [comment]
@wytseoortwijn 
Infer-modifies flag is something related to Boogie backend files. Used only in one example 
sequential->BoogieWithGlobals.java
[Reference Paper (page08)](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/specsharp-krml160.pdf)

**Comment by wytseoortwijn** [comment]
Fixed in #138.
